### ISSUE TITLE
Neural networks not training

### ISSUE THREAD

**Issue by generic-github-user** [generic-github-user/xkcd-Generator#8]

**Comment by generic-github-user** [comment]
Could both models be training at once where only one is supposed to? Possible solutions:

- Create a custom model with tensor ops and use [varList](https://js.tensorflow.org/api/0.11.7/#class:train.Optimizer) to train only certain weight layers

- Retrieve specific variable names from models and use varList to train only those variables

- Evaluate one model before passing its outputs to the next model

**Comment by generic-github-user** [comment]
Testing has confirmed that the discriminator network trains well; however, the generator network is almost impossible to train.

**Comment by generic-github-user** [comment]
Generator network does not actually train; loss value not changing when discriminator training function is disabled.

**Comment by generic-github-user** [comment]
Generator network loss completely minimized when varList is disabled.

**Comment by generic-github-user** [comment]
The discriminator network _does_ train when its trainable variables ([varList](https://js.tensorflow.org/api/latest/index.html#tf.train.Optimizer.minimize)) are defined, unlike the generator network. This indicates that the trainable variables are correctly defined, so something else must be preventing the generator network from training.

**Comment by generic-github-user** [comment]
Labels for [tf.losses.logLoss()](https://js.tensorflow.org/api/latest/index.html#losses.logLoss) function were placed as the second argument, instead of the first. Resolving this issue did not affect the performance of the training algorithm.

**Comment by generic-github-user** [comment]
Generator network now briefly training with discriminator network training disabled, but output values quickly explode:
```
Tensor
     [[9697017987072, 12064277921792, 0, ..., 0, 0, 10895708127232],]
```
The generator network likely tricks the discriminator network (which has not yet learned this characteristic of generated data) using arbitrarily large values.

**Comment by generic-github-user** [comment]
~~I think it's fixed.~~ Decreased learning rate for adam optimizer used to train the generator network. Pixel levels are still inaccurate (dark images are generated, most training images are mostly white). Even when tested on one training image (to test copying of training data), this error still occurs.

![Screenshot](https://raw.githubusercontent.com/generic-github-user/xkcd-Generator/master/testing/14.PNG)
